src/core/datastore/repositories/storage-type.js

```diff
@@ -4,5 +4,6 @@ export const storageType = {
   webSql: 'webSql',
   indexedDb: 'indexedDb',
   localStorage: 'localStorage',
-  sessionStorage: 'sessionStorage'
+  sessionStorage: 'sessionStorage',
+  sqlite: 'sqlite'
```

Comment:
Is there a way to make the storage types specific to the platform being used?

Reply:
The enum isn't public yet. I didn't plan on making the enum values the public way of checking what is supported, I thought we'd have an official way, an API (like a method or something). I'd like to reduce the "branching" of structures for each shim. As I said in another comment, this part isn't "officially" implemented so I tried to not go into too much detail. This can be decided as part of the "tweaking" ticket to make the two implementations "meet" :)